lib/dl_file_uploader_worker_lib/dl_file_uploader_worker_lib/tasks/excel.py

```diff
@@ -95,63 +114,109 @@ async def run(self) -> TaskResult:
                 raw_schema: list[SchemaColumn] = []
                 raw_schema_header: list[SchemaColumn] = []
                 raw_schema_body: list[SchemaColumn] = []
-                source_title = f"{dfile.filename} – {sheetname}"
-                sheet_data_source = DataSource(
-                    title=source_title,
-                    raw_schema=raw_schema,
-                    status=source_status,
-                    error=None,
-                )
-                dfile.sources.append(sheet_data_source)
+
+                if self.meta.exec_mode == TaskExecutionMode.BASIC:
+                    source_title = f"{dfile.filename} – {sheetname}"
+                    if dfile.file_type == FileType.yadocs:
+                        sheet_data_sources = [
+                            DataSource(
+                                title=source_title,
+                                raw_schema=raw_schema,
+                                status=source_status,
+                                user_source_dsrc_properties=YaDocsUserSourceDataSourceProperties(sheet_id=sheetname),
```

Comment:
`sheet_id=sheetname`
Well, this is confusing))
If you don't like to call `sheet_title`, let's at least leave a comment here and in the connection class, that sheet_id is actually a title of a sheet)